### DISCUSSION TITLE
Hosted hyperdrive updates

### DISCUSSION THREAD

**Discussion by Vort** [beakerbrowser/beaker#1875] [Q&A]
Let's say someone created hyperdrive with a simple webpage.
10 users decided to help author with the hosting.

Next author decided to add large file to his website.
What will happen next?
Will it be automatically copied to 10 users?
If so, they may run out of space unexpectedly and even don't know why.

If not, then how user will be able to tell software his decision - whether he want to continue hosting of the new version or not?

**Comment by pfrazee** [comment]
This is accurate and we haven't really figured out the ideal way to answer it. I can imagine putting policies into play like "host up to X bytes of this hyperdrive."
